amr-wind/physics/HybridRANSLESABL.cpp

```diff
@@ -7,6 +7,7 @@
 namespace amr_wind {
 namespace hybrid_rans_les_abl {
 
+// cppcheck-suppress uninitMemberVar
 HybridRANSLESABL::HybridRANSLESABL(const CFDSim& sim) : m_sim(sim) {}
```

Comment:
is it complaining that m_tke and m_sdr are not initialized? we could set them to nullptr in the header?